src/llm_vicuna.py

```diff
@@ -61,9 +61,10 @@ def download_model():
         "ignore", category=UserWarning, message="TypedStorage is deprecated"
     )
 
-    # This version of FastChat hard-codes a relative path for the model ("./model"),
-    # making this necessary :(
-    os.chdir("/FastChat")
+    if os.path.exists("/FastChat"):
```

Comment:
@erikbern I'm not sure how to add this in a path such that it exists in this function call, open to suggestions